packages/polytechcolloscopeclient/lib/src/consts.dart

```diff
@@ -0,0 +1,37 @@
+enum Year {
+  first,
+  second,
+}
+
+class Consts {
+  static const String userAgent =
```

Comment:
Is it really needed?

Reply:
No but i copy pasted it along with the base url from the cas library x)